src/main/java/kb/weather/exceptions/ExceptionHandlerController.java

```diff
@@ -0,0 +1,22 @@
+package kb.weather.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+
+@ControllerAdvice
+public class ExceptionHandlerController {
+    @ExceptionHandler(EmptyInputException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public void handleEmptyInputException(EmptyInputException e) {
+        System.out.println(e.getMessage());
```

Comment:
you can use `@Slf4j` annotation on the ExceptionHandlerController and `log.warn();`